tasks/readme.rake

```diff
@@ -1,6 +1,12 @@
 # typed: true
 # frozen_string_literal: true
 
+# Temporary fix for CI
+# A patch as been upstreamed, we're waiting for it to be merged.
+# See https://github.com/rubyconfig/config/pull/319
+# TODO: remove this one the patch is released.
+require "rails"
```

Comment:
This is a patch to help fix CI that will be merged soon (#965). Once it's merged I'll rebase on main.